src/Command/Binary/ReadBuffer.php

```diff
@@ -161,12 +161,16 @@ protected function handleBody($buffer)
     {
```

Comment:
Maybe we should split `handleBody` in 2 sections: handleArgs() and handleData() (since data is always the last argument). 

handleBody:
- Consume arguments and possible right-padded \0s until the last argument is consumed or we hit data
- If we hit data: consume all reamining bytes as data
- Check if all consumed bytes sum up to the specified body-length